apps/cli/src/lib/ensure-version.ts

```diff
@@ -1,24 +1,51 @@
-// eslint-disable-next-line @nx/enforce-module-boundaries
-import * as getJson from 'get-json';
+import * as https from 'https';
 import { Color, color } from './log-colors';
 
 const red = color(Color.FgRed);
 const yellow = color(Color.FgYellow);
 
 async function getLatestVersion(pkgName: string) {
-  return getJson(`https://registry.npmjs.org/${pkgName}`).then(
-    (packageData: { 'dist-tags': { latest: string } }) => {
-      return packageData['dist-tags'].latest;
-    }
-  );
+  const url = `https://registry.npmjs.org/${pkgName}`;
+  return new Promise((resolve, reject) => {
+    const options = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const request = https.get(url, options, (response) => {
+      const chunks = [] as string[];
```

Comment:
`as` forces the type.

```suggestion
      const chunks: string[] = [];
```